components/aws/sagemaker/common/_utils.py

```diff
@@ -327,12 +330,16 @@ def create_endpoint_config(client, args):
     for key, val in args['endpoint_config_tags'].items():
         request['Tags'].append({'Key': key, 'Value': val})
 
+    return request
+
+def create_endpoint_config(client, args):
+    request = create_endpoint_config_request(args)
     try:
         create_endpoint_config_response = client.create_endpoint_config(**request)
         logging.info("Endpoint configuration in SageMaker: https://{}.console.aws.amazon.com/sagemaker/home?region={}#/endpointConfig/{}"
-            .format(args['region'], args['region'], endpoint_config_name))
+            .format(args['region'], args['region'], request['EndpointConfigName']))
         logging.info("Endpoint Config Arn: " + create_endpoint_config_response['EndpointConfigArn'])
-        return endpoint_config_name
+        return request['EndpointConfigName']
```

Comment:
Why this change ? Some caller might need `endpoint arn`.

Reply:
nothing changed 
request['EndpointConfigName'] = endpoint_config_name